subdoc/lib/parse_comment.h

```diff
@@ -159,7 +159,8 @@ inline sus::Result<ParsedComment, ParseCommentError> parse_comment(
             // Empty line, preserve the paragraph break.
             parsed << "</p><p>";
             add_space = false;
-          } else if (s.starts_with("#")) {
+          } else if (s.starts_with("#") && !inside_pre &&
```

Comment:
imo `s.starts_with("#") && !(inside_pre || inside_code)` reads easier to me your call.

Reply:
Good idea.